batchflow/models/torch/mixins.py

```diff
@@ -140,6 +132,85 @@ def show_lr(self):
         plt.grid(True)
         plt.show()
 
+    def show_loss(self, overlay_lr=True, figsize=(12, 6), window=20, final_window=50, minor_tick_frequency=4,
+                  log_loss=False, log_lr=False, return_figure=False, savepath=None, save_kwargs=None):
+        """ Plot loss and learning rate over the same figure.
+
+        Parameters
+        figsize : tuple of ints
+            Size of the figure.
+        window : int or None
+            If int, then averaged graph of loss values is displayed over the regular one.
+            The average for each point is computed as the mean value of the kernel with the center in this point.
```

Comment:
Why center, not previous points? THis will skew the plot at the edges.

Reply:
The moving average will be always somewhat skewed, to one side or to the other. Using the previous points makes the `average_loss` lag noticeably, which looks unnatural for most actual loss functions. Current implementation makes edges a bit smoother and with less wiggles, which is not as bad.